setup.py

```diff
@@ -76,8 +74,6 @@
         'Operating System :: Unix',
         'Operating System :: Microsoft :: Windows',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```

Comment:
Do we need to add 3.7 and 3.8 here

Reply:
3.7 and 3.8 are there. I don't know why they don't show in the diff